### DISCUSSION TITLE
Hacktoberfest topic

### DISCUSSION THREAD

**Discussion by Andreas02-dev** [topgrade-rs/topgrade#962] [General]
Hi,

Perhaps it would be good to add the Hacktoberfest topic to the repository for this month.
That way, more people might discover and/or contribute to Topgrade.

More information can be found here: https://github.com/topics/hacktoberfest

@SteveLauC, you've got ownership permissions to add the topic, right?

Let me know your thoughts.

With kind regards,
Andreas

**Comment by SteveLauC** [comment]
Hi, thanks for bringing this to our attention!

> Each October, open source maintainers give new contributors extra attention as they guide developers through their first pull requests on GitHub.


Though I am not sure if I could be more active on the project this month, so...

But anyway, thanks for letting me know this!
